homeassistant/components/sensor/buienradar.py

```diff
@@ -167,15 +191,25 @@ def entity_picture(self):
 
         if self.type != SYMBOL:
             return None
-        else:
-            return self._entity_picture
+
+        return self._entity_picture
 
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        from buienradar.buienradar import (PRECIPITATION_FORECAST)
+
+        if self.type.startswith(PRECIPITATION_FORECAST):
+            result = {ATTR_ATTRIBUTION: self._attribution}
+            if self._timeframe is not None:
+                result[TIMEFRAME_LABEL] = "%d min" % (self._timeframe)
+
+            return result
+
         return {
             ATTR_ATTRIBUTION: self._attribution,
             SENSOR_TYPES['stationname'][0]: self._stationname,
+            UPDATED: self._measured,
```

Comment:
There is no need for this attribute. Each state already has an `updated` attribute, which is updated automatically when new information is published to the state machine.